_docs/tasks/metrics-logs.md

```diff
@@ -9,210 +10,300 @@ type: markdown
+    outputAsJson: true
+   ---
+   # Rule to send logentry instances to a stdio handler
+   apiVersion: "config.istio.io/v1alpha2"
+   kind: rule
+   metadata:
+     name: newlogstdio
+     namespace: istio-config-default
+   spec:
+     match: "true" # match for all requests
+     actions:
+      - handler: newhandler.stdio
+        instances:
+        - newlog.logentry
+   ---
    ```
 
-   The expected output is:
+1. Push the new configuration.
 
    ```bash
-   Error: the server could not find the requested resource
+   kubectl -n istio-config-default apply -f new_telemetry.yaml
    ```
 
-   If your selected service has service-specific rules, update `new_rule.yaml`
-   to include the existing rules appropriately. Append the rule from `new_rule.yaml`
-   to the existing `rules` block and save the updated content back over `new_rule.yaml`.
-
-1. Push the new configuration to Mixer for a specific service.
-
-   ```bash
-   istioctl mixer rule create reviews.default.svc.cluster.local reviews.default.svc.cluster.local -f new_rule.yaml
+   Expected output:
+   ```
+   metric "doublerequestcount" configured
+   prometheus "doublehandler" configured
+   rule "doubleprom" configured
+   logentry "newlog" configured
+   stdio "newhandler" configured
+   rule "newlogstdio" configured
    ```
 
-1. Send traffic to that service.
+1. Send traffic to the sample application.
 
-   For the BookInfo sample, visit `http://$GATEWAY_URL/productpage` in your web browser or
-   issue the following command:
+   For the BookInfo sample, visit `http://$GATEWAY_URL/productpage` in your web
+   browser or issue the following command:
 
    ```bash
    curl http://$GATEWAY_URL/productpage
    ```
 
-   For purposes of this task, please refresh the page several times or issue the curl
-   command a few times to generate traffic.
+   For purposes of this task, please refresh the page several times or issue the
+   curl command a few times to generate traffic.
+
+1. Verify that the new metric values are being generated and collected.
+
+   Setup port-forwarding for Prometheus:
 
-1. Verify that the new metric is being collected.
-   
-   Setup port-forwarding for Grafana:
-   
    ```bash
-   kubectl port-forward $(kubectl get pod -l app=grafana -o jsonpath='{.items[0].metadata.name}') 3000:3000 &
+   kubectl -n istio-system port-forward $(kubectl -n istio-system get pod -l app=prometheus -o jsonpath='{.items[0].metadata.name}') 9090:9090 &
    ```
 
-   Then open the Istio dashboard in a web browser: [http://localhost:3000/dashboard/db/istio-dashboard](http://localhost:3000/dashboard/db/istio-dashboard)
-   
-   One of the rows in the dashboard will be named "reviews". If that row is not visible, please refresh the dashboard page. The "reviews" row
-   contains a graph entitled "Response Size by Source And Version". The graph displays a breakdown of the distribution of Response Sizes returned
-   by the "reviews" service.
+   Then open [this
```

Comment:
You can now view the number of requests via this link.. 
add explanation as another sentence.

Reply:
Added some text that hopefully clarifies.